src/widgetastic_patternfly/__init__.py

```diff
@@ -2351,3 +2351,111 @@ def item_select(self, item, close=True):
         finally:
             if close:
                 self.close()
+
+
+class ItemsList(View):
+    """Basic list-view handling class:
+    https://www.patternfly.org/pattern-library/content-views/list-view/
+    Most functionality is meant to mimic widgetastic's Table class.
+    This class is meant to work with an ItemClass that is defined elsewhere. The ItemClass
+    is a ParametrizedView that represents a single item in the list-view
+    (similar to TableRow and Table). The parameter for the ItemClass is it's index/position in
+    the list-view.
+
+    In practice, the ItemClass will be defined within a nested view e.g. in a view class we could
+    have:
+    .. code-block:: python
+        # define a view
+        class MyView(BaseLoggedInPage):
+            # define the list-view widget that's on this page
+            @View.nested
+            class item_list(ItemsList):
+                # define the item_class that the list-view uses
+                item_class = ItemClass
+            # define whatever else this page has on it ...
+
+    For an example: integration_tests/cfme/control/explorer/alerts.py::MonitorAlertsAllView
+
+    Usage is as follows assuming the list is instantiated as ``view.item_list``:
+
+    .. code-block:: python
+        # Access item by position
+        view.item_list[0].next() # => gives first item in list-view
+        # Iterate through rows
+        for item in view.item_list:
+            do_something()
+        # You can also filter items in two main ways
+        # 1) by an assoc_column that corresponds to an attribute of the item_class e.g.
+        item_list = view.item_list
+        item_list.assoc_column = 'description'
+        filtered_items = item_list[<desired_description>] # where <desired_description> is a str
+        # 2) by key-value pairs
+        item_filter = {'description': <desired_description>}
+        filtered_items = view.item_list[item_filter]
+        # note that filters can also be applied to the items() method
+        # e.g.
+        filtered_items = view.item_list.items(item_filter)
+
+    Args:
+         assoc_column: Name of an attribute/property defined in the item_class
+    """
+
+    ROOT = './/div[contains(@class,"list-view-pf-view")]'
```

Comment:
ROOT locator should not be relative xpath expression, view instantiation should handle setting the parent browser object so that it is correctly located within the parent View (if this is nested in a main content div view)